platforms/winrt/readme.txt

```diff
@@ -119,3 +119,41 @@ To generate Windows Phone 8.1 x86 project files in the opencv/bin dir
 mkdir bin
 cd bin
 cmake -G "Visual Studio 12 2013" -DCMAKE_SYSTEM_NAME=WindowsPhone -DCMAKE_SYSTEM_VERSION=8.1 ../
+
+Running tests for Windows Store
+===============================
+1. You might need to install this if you haven't already: http://www.microsoft.com/en-US/download/details.aspx?id=40784
+
+2. Set OPENCV_TEST_DATA_PATH environment variable to location of opencv_extra/testdata (cloning of https://github.com/Itseez/opencv_extra repo required) to get tests work correctly. Also, set OPENCV_PERF_VALIDATION_DIR environment variable in case you are planning to build and run performance tests.
```

Comment:
`OPENCV_PERF_VALIDATION_DIR` is **not required** to succesfully run performance tests.
It is additional check for changed performance results.